stress/common/src/main/java/alluxio/stress/master/MasterBenchTaskResultStatistics.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.master;
+
+import alluxio.Constants;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.HdrHistogram.Histogram;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.zip.DataFormatException;
+
+/**
+ * Statistics class that is used in {@link MasterBenchTaskResult}.
+ */
+public class MasterBenchTaskResultStatistics {
+  public static final int MAX_RESPONSE_TIME_COUNT = 20;
+
+  /** The response time histogram can record values up to this amount. */
+  public static final long RESPONSE_TIME_HISTOGRAM_MAX = Constants.SECOND_NANO * 60 * 30;
+  public static final int RESPONSE_TIME_HISTOGRAM_PRECISION = 3;
+
+  static final int COMPRESSION_LEVEL = 9;
+  static final int RESPONSE_TIME_99_COUNT = 6;
+
+  public long mNumSuccess;
```

Comment:
Should there be getters and setters for these, for json ser/de?

Reply:
These are currently all public for json ser/de.